pkgs/top-level/all-packages.nix

```diff
@@ -3030,6 +3030,7 @@ with pkgs;
       UniformTypeIdentifiers
       UserNotifications
     ;
+    buildGoModule = buildGo122Module;
```

Comment:
Maybe we can move this into the file to easier keep it in sync with the go used there

Reply:
I just did as in other packages but yes I also prefer to move it.